src/js/toolbar.js

```diff
@@ -356,6 +356,7 @@ var Toolbar;
         showAndUpdateToolbar: function () {
             this.modifySelection();
             this.setToolbarButtonStates();
+            this.base.trigger('updateToolbar', {}, this.base.getFocusedElement());
```

Comment:
probably? i feel like there was a compelling case for putting it exactly where it is (this is a direct monkey-patch we've been carrying around locally for the dynamic-button functionality we created), though I couldn't defend the decision as it was made long ago.